### DISCUSSION TITLE
Alternative geocoding service

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#1149] [Ideas]
The stargazers plugin now supports a map visualisation which looks really good.  However I would rather not set up a Google Billing account for it.

I think it would be useful to add support for alternative services, especially OpenStreetMap Nominatim which has lots of [providers](https://wiki.openstreetmap.org/wiki/Nominatim#Alternatives/Third-party_providers) some with reasonable free daily limits, including [MapQuest](https://developer.mapquest.com/documentation/open/nominatim-search/) which has a free limit of [15,000 per month](https://developer.mapquest.com/plans).  The base url endpoint could be a setting so that the user can choose which provider works best for them.  Most [JS Nominatim clients](https://www.npmjs.com/search?q=nominatim) support this.

**Comment by lowlighter** [comment]
Yeah, without prepaid/virtual cards, setting a Google billing account could be kind of dangerous x)

I'll check the alternatives you provided when I have some free time, or if someone else is interested to look into it 👍
